src/fragments/start/getting-started/ionic/data-model.mdx

```diff
@@ -116,41 +102,76 @@ Note: If you are using Angular 6 or above, you may need to add the following to
 };
 ```
 
-In your `src/app/app.component.ts` file, add the following imports and modifications to your class to to add data to your database with a mutation by using the `API.service` file which was generated when you ran `amplify add api`:
+First, enable the Angular forms modules in `src/app/app.module.ts`:
```

Comment:
Add a new setting up the `AmplifyAuthenticatorModule` section here

Reply:
I might be misunderstanding, but we're not adding Angular form module here, right? We're adding `AmplifyAuthenticatorModule`. Should it be something like 

```suggestion
First, enable `AmplifyAuthenticatorModule` in `src/app/app.module.ts`:
```
?

Reply:
Good catch I'll fix.

Reply:
469f9888dd096140f038b6dcfd89246eb97e8478